packages/onboarding-ui/src/forms/OrganizationInfoForm/OrganizationInfoForm.tsx

```diff
@@ -165,9 +170,20 @@ const OrganizationInfoForm = ({
           <Button onClick={onBackButtonClick}>
             {t('component.form.action.back')}
           </Button>
+
           <Button type='submit' primary disabled={isValidating || isSubmitting}>
-            {t('component.form.action.next')}
+            {onConfirmText || t('component.form.action.next')}
```

Comment:
Why not `??`?

```suggestion
            {onConfirmText ?? t('component.form.action.next')}
```

It's odd to imagine a valid falsy value here, but...